### DISCUSSION TITLE
How can I suggest edits to the documentation?

### DISCUSSION THREAD

**Discussion by jsquaredosquared** [reflex-dev/reflex#587] [Q&A] [answered]
I found a few typos while reading through the documentation and wanted to notify someone to fix it.  For example, in the "Creating a substates" section of the "Substates" page "inherit" is misspelled as "inheret".

**Comment by Alek99** [comment, marked as answer]
Feel free to make an issue or put a pr up here https://github.com/pynecone-io/pcweb

**Reply by jsquaredosquared** [reply]
Thanks
